adapters/ix/ix.go

```diff
@@ -359,3 +363,33 @@ func BuildIxDiag(request *openrtb2.BidRequest) error {
 	}
 	return nil
 }
+
+// moves sid from imp[].ext.bidder.sid to imp[].ext.sid
+func moveSid(imps []openrtb2.Imp) error {
+	if len(imps) < 1 {
+		return nil
+	}
+	imp := imps[0]
```

Comment:
Could you please help in explaining the reason for just using the first impression to move sid? What if there are multiple impressions in the request object? I think you might wanna use a for loop.